docs/commands/index.asciidoc

```diff
@@ -0,0 +1,13 @@
+[[commands]]
```

Comment:
Because there is only one command-line topic, I don't feel this top-level topic is needed.  How about moving the kibana-encryption-keys  doc to either the Setup or Stack Management > Security section and using the title:

Set up encryptions keys to protect sensitive information

Reply:
Fixed with c0bf78f